packages/arb-token-bridge-ui/src/components/TransferPanel/DepositCardL2Failure.tsx

```diff
@@ -39,17 +85,17 @@ export function DepositCardL2Failure({ tx }: { tx: MergedTransaction }) {
           onClick={() => redeem(tx)}
           className="text-2xl"
         >
-          Re-execute
+          Retry for free
         </Button>
       </Tooltip>
 
       <div className="h-2" />
       <div className="flex flex-col font-light">
         <span className="text-lg text-orange-dark">
-          L1 transaction: <DepositL1TxStatus tx={tx} />
+          L1 transaction (Etherscan): <DepositL1TxStatus tx={tx} />
```

Comment:
We can remove the name of the explorer, it's from an outdated design.

Reply:
Done. Updated according to the new design.